### DISCUSSION TITLE
Need help getting started with MD > Live Zotero Citations

### DISCUSSION THREAD

**Discussion by kerim** [retorquere/zotero-better-bibtex#2873] [Q&A] [answered]
Dear Community,

I need help getting started with the following workflow. 

1. I start by writing in Obsidian, using Pandoc Reference List to insert Zotero citations into my document. This works with BetterBibTex, and looks like this:

`"quoted text" @wolfeHistoricisingVoiceGod1997 [p150].`

2. I can then use Pandoc Export to export this file to markdown on my desktop.

So far so good, I think? But then I'm stuck. 

3. I then want to scan the document with BetterBibTex to create a Word or Open Document file that has "live" Zotero citations in it. That is, that will work with the Zotero plugin within Word, Open Office, or Google Docs. 

So far the only documentation I can find is [this page](https://retorque.re/zotero-better-bibtex/exporting/pandoc/#from-markdown-to-zotero-live-citations) but this is a bit obscure, complicated for me. I can't even select a markdown file from the BetterBibtex menu in Zotero - all the files are greyed out. Based on [this discussion](https://github.com/retorquere/zotero-better-bibtex/discussions/2671), it seems my problem might be that I don't have the path to pandoc listed in the YMAL? But how to find that? Is there something else I need here as well.

Perhaps what I need is a very specific example of what the YMAL header should look like? I've never made one before. Mine currently looks like this (trying to guess from the documentation):

```
---
zotero:
  scannable-cite: true
  author-in-text: false 
  csl-style: Chicago Manual of Style 17th edition (author-date) 
  sorted: true 
---

My document text here..

```

**Comment by retorquere** [comment]
The yaml front matter is optional, it will work without it, but you will need to run pandoc from the command line to get the document with live citations.

**Comment by kerim** [comment]
Thanks. Then I need help doing that. So far the only step-by-step guide I've found is [this one](https://forums.zotero.org/discussion/comment/407793/#Comment_407793). Hopefully it isn't out of date, though it does look far more complex than what I had hoped for.

**Reply by retorquere** [reply, marked as answer]
The majority of the steps outlines there are one-time setup things tied to particular preferences in editing the markdown document, not converting it, and there's no need (no point really) in exporting the library as done in step 4d, because it isn't used in the process. You just have a markdown file, the yaml header entirely optional, with citation keys matching the ones in Zotero, and run

```
pandoc --lua-filter=zotero.lua test.md -s -o success.docx 
```

and that will get you a docx file with live references. If you prefer a GUI, there may be GUIs out there for running pandoc, but I'm not familiar with them.

**Comment by kerim** [comment]
I got it to work. Thanks. Actually, there were some problems with my workflow, but they don't have to do with Betterbibtex. If I sort everything out I'll try to write up my own documentation for others like me...

**Comment by kerim** [comment]
I've written my own how-to guide here:

https://garden.oxus.net/technology/obsidian-to-word/

**Reply by retorquere** [reply]
The export is unnecessary in that setup

**Reply by kerim** [reply]
I've updated the guide. Thanks.
